docs/how-tos/deploy-ml.md

```diff
@@ -207,17 +377,27 @@ This service will deploy and run the model.
 {{<imgproc src="/services/icons/vision.svg" class="fill alignleft" style="max-width: 150px" declaredimensions=true alt="Configure a service">}}
 **2. Configure an <code>mlmodel</code> vision service**
 
-The vision service takes the the ML model and applies it to the stream of images from your camera.
+The vision service takes the ML model and applies it to the stream of images from your camera.
 
 Add the `vision / ML model` service to your machine.
 Then, from the **Select model** dropdown, select the name of the ML model service you configured in the last step (for example, `mlmodel-1`).
 
+Click **Save** to save your changes.
+
 {{% /tablestep %}}
 {{% tablestep link="/services/vision/mlmodel/#test-your-detector-or-classifier" %}}
 {{<imgproc src="/services/ml/deploy.svg" class="fill alignleft" style="max-width: 150px" declaredimensions=true alt="Deploy your model">}}
-**3. Test your classifier**
+**3. Use your detector or classifier**
```

Comment:
Did it use to be the case that only classifiers were supported on the test panel, but now it can do both?

Reply:
this guide used to be specific to classifiers. I think what you said may have also been true but I don't remember the details